website/docs/riverpod_for_provider_users.mdx

```diff
@@ -18,62 +18,179 @@ import {
 This article is designed for people familiar with the [Provider] package who
 wants to learn about Riverpod.
 
-## The relationship between Riverpod and Provider
+Since Provider is widely popular, why would one migrate to Riverpod?
```

Comment:
That probably should be a heading instead of the current "Provider's limitations"